tests/e2e/utils/createNewStory.js

```diff
@@ -18,12 +18,22 @@
  * WordPress dependencies
  */
 import { visitAdminPage } from '@wordpress/e2e-test-utils';
+/**
+ * Internal dependencies
+ */
+import { getTimeTracker } from '../../../assets/src/tracking/trackTiming';
 
 /**
  * Creates a new story.
  */
 async function createNewStory() {
+  const trackTiming = getTimeTracker(
+    'visitAdminPage',
+    'createNewStory',
+    'Publish Story'
+  );
   await visitAdminPage('post-new.php', 'post_type=web-story');
+  trackTiming();
```

Comment:
Can you explain this change here?

I don't think we want to call this in our E2E test utils? 🤔  It would not work either because it's not running in the browser context.

Reply:
Ah, got it, just carefully read the context and this is refer to the testing part, removed.